ote_sdk/ote_sdk/tests/parameters_validation/validation_helper.py

```diff
@@ -0,0 +1,23 @@
+"""
+Common functions for input parameters validation tests
+"""
+
+# Copyright (C) 2021-2022 Intel Corporation
+# SPDX-License-Identifier: Apache-2.0
+#
+
+import pytest
+
+
+def check_value_error_exception_raised(
+    correct_parameters: dict, unexpected_values: list, class_or_function
```

Comment:
Callable?

Reply:
thanks, added "Callable" annotation for class_or_function parameter